### DISCUSSION TITLE
How to use [UseFiltering] and [UseSorting] on queries? Doesn't seem to work.

### DISCUSSION THREAD

**Discussion by ghost** [ChilliCream/hotchocolate#3632] [Q&A]
Is this the right way to apply `[UseFiltering, UseSorting]` attributes?

**Step 1 Startup.cs**
```csharp
services.AddGraphQLServer()
  // add these lines
  .AddFiltering()
  .AddSorting();
```

Using attribute [UseFiltering] and applying to query does not filter the data.

**Comment by PascalSenn** [comment]
Hi @bachmannn 

Can you provide a repro? This should work as it is. Maybe it's a configuration issue
